components/api_server/src/quality_time_server.py

```diff
@@ -5,22 +5,24 @@
 monkey.patch_all()
 
 import logging
-import os
 
 import bottle
 
+from shared.utils.env import getenv, loadenv
+
 from initialization.database import init_database
 from initialization.bottle import init_bottle
 
 
 def serve() -> None:  # pragma: no feature-test-cover
     """Connect to the database and start the application server."""
-    log_level = str(os.getenv("API_SERVER_LOG_LEVEL", "WARNING"))
+    loadenv(".env", "default.env")
```

Comment:
I don't think we should load the `default.env` which contains example values

Reply:
They are not example values, they are the default values for the environment variables.

Reply:
The LDAP settings contain: `LDAP_ROOT_DN=dc=example,dc=org`